src/components/UriViewer/UriViewer.js

```diff
@@ -49,8 +49,8 @@ export const UriViewer = _.flow(
         setAzureStorage(metadata);
       } else {
         // TODO: change below comment after switch to DRSHub is complete, tracked in ticket [ID-170]
```

Comment:
Oh can you remove this part of the comment and mark [ID-170] as done? It was just to update this comment after switching to drshub for signed urls